kornia/contrib/extract_patches.py

```diff
@@ -329,19 +326,21 @@ def extract_tensor_patches(
     if len(input.shape) != 4:
         raise ValueError(f"Invalid input shape, we expect BxCxHxW. Got: {input.shape}")
 
+    window_size = _pair(window_size)
+    stride = _pair(stride)
+
+    if (stride[0] > window_size[0]) | (stride[1] > window_size[1]):
+        raise AssertionError(f"Stride={stride} should be less than or equal to Window size={window_size}")
+
```

Comment:
In the event `stride` is greater than `window_size`, we need to throw an error since info is lost and can't be restored.